data-es2016plus.js

```diff
@@ -882,7 +882,11 @@ exports.tests = [
           ie11: false,
           edge15: edge.experimental,
           edge16: true,
-          edge17: false,
+          edge17: {
+            val: false,
+            note_id: 'edg-shared-memory-spectre',
+            note_html: 'The feature was temporarily disabled to mitigate the Meltdown and Spectre CPU bugs</a>.',
```

Comment:
Just noticed there is only closing `</a>` tag without opening `<a>` tag

Reply:
My bad.  I didn't notice it when I copied the note from Chrome.  Fixed both on #1295 .